src/odemis/driver/test/pvcam_test.py

```diff
@@ -55,18 +63,7 @@ class TestPVCam(VirtualTestCam, unittest.TestCase):
 
     @classmethod
     def setUpClass(cls):
-        if not TEST_NOHW:
-            super(TestPVCam, cls).setUpClass()
-
-    @classmethod
-    def tearDownClass(cls):
-        if not TEST_NOHW:
-            super(TestPVCam, cls).tearDownClass()
-
-    def setUp(self):
-        if TEST_NOHW:
-            self.skipTest("No simulator available")
-        super(TestPVCam, self).setUp()
+        raise unittest.SkipTest("No simulator available")
```

Comment:
will we always skip this test? Don't we need to define an if statement? Even if it is checking a constant defined at the top of the file. If you want to run the tests on a 32 bit system you just need to adjust one variable instead of commening two setUpClass methods.

Reply:
Indeed, my bad. I was so enthusiastic of removing the code, that I forgot we should still run the code if the hardware is available!

=> I've updated with a `if TEST_NOHW`.